sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,4 +1,20 @@
 # Change Log azure-data-appconfiguration
+## Version 1.0.0-preview.6 (2019-10-25)
```

Comment:
There are two 1.0.0-preview6 headings in the same file.  Can you merge these additions with the section that already exists right below?  Also let's double-check that aka.ms link, I think it would be different (it won't match the AzConfig package version exactly).

/cc @hemanttanwar @loarabia